src/style.css

```diff
@@ -1 +1,81 @@
 /* Styles go here */
+body {
+  font-family: "Roboto", sans-serif;
+  background-color: aliceblue;
+  margin: 0;
+  padding: 0;
+}
+
+.logo {
+  display: inline-block;
+  text-decoration: none;
+  margin-top: 2px;
+  padding: 0 20px;
+}
+
+.header {
+  box-sizing: border-box;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  padding: 0 30px;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.nav {
+  position: relative;
+}
+
+.nav_menu {
+  display: inline-flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_list {
+  list-style: none;
+}
+
+.nav_link {
+  position: relative;
+  display: block;
+  font-family: "Roboto", sans-serif;
```

Comment:
just set font-family to html selector, so other elements will inherit it

Reply:
* Not fixed: